mxnet/training/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -110,10 +110,8 @@ RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSIO
  && rm -rf /var/lib/apt/lists/*
 
 
-# MXNet requires pip 19.3.1 due to being backwards compatible
-# with python2
 RUN ${PIP} --no-cache-dir install --upgrade \
-    pip==19.3.1 \
+    pip \
```

Comment:
Does the version of MXNet in this container no longer require a pip compatible with python2? I'm not sure we have tests for this. It's probably safe, but good to verify. pip 21.0 removed python2 support, latest version is 21.1.2

Reply:
no, latest pip is fine,  I have confirmed internally